notebooks/en/labelling_feedback_setfit.ipynb

```diff
@@ -0,0 +1,2842 @@
+{
```

Comment:
<p>might be nice to add a link to the dataset here? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>